src/site/_includes/spaces.njk

```diff
@@ -34,13 +34,19 @@ layout: default-next
       {% endfor %}
     </ul>
     <time class="eyebrow">{{ space.data.event_date | prettyDate('DATE_FULL') }}</time>
+    <p class="eyebrow"> {{ space.data.event_time }} </p>
     <p class="gap-top-size-1 gap-bottom-size-2">{{ space.data.description }}</p>
   </div>
   {% if space.data.twitter_link %}
     <a class="button" href="{{ space.data.twitter_link }}" target="_blank">
       {% include "icons/twitter.svg" %}
       <span>{{ 'i18n.spaces.see_on_twitter' | i18n(locale) }}</span>
     </a>
+  {% elif space.data.cal_link %}
```

Comment:
Ideally, the `calendarLink()` filter would be flexible enough to take in a time of day as well as the date, and it could replace this hardcoded `space.data.cal_link`.